File: app/src/main/java/com/viafourasample/src/managers/ColorManager.java
package com.viafourasample.src.managers;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

import com.viafourasample.src.model.SettingKeys;

public class ColorManager {
    public static boolean isDarkMode(Context context){
        return PreferenceManager.getDefaultSharedPreferences(context).getBoolean(SettingKeys.darkMode, false);
    }
}

File: app/src/main/java/com/viafourasample/src/managers/ColorManager.kt
package com.viafourasample.src.managers

import android.content.Context
import android.preference.PreferenceManager
import com.viafourasample.src.model.SettingKeys

object ColorManager {
    fun isDarkMode(context: Context): Boolean =
        PreferenceManager.getDefaultSharedPreferences(context)
            .getBoolean(SettingKeys.darkMode, false)
}
